pkg/wireguard/agent/agent.go

```diff
@@ -236,21 +244,35 @@ func (a *Agent) UpdatePeer(nodeName, pubKeyHex string,
 	}
```

Comment:
line 240 should be `option.Config.EnableIPv4 && nodeIPv4 != nil`. Similar changes for line 242